### DISCUSSION TITLE
I just want to share some token value from one request to another

### DISCUSSION THREAD

**Discussion by emartinezagra** [usebruno/bruno#1576] [Q&A]
Hi . One of my requests is returning  some registerID value, that I want to pass to other requets. 
I need to run everything from CLI
So my REQUEST1 Post Response script has this 

```
bru.setEnvVar("registerId", res.body.data.registerId);
console.log("registerId set to :"+bru.getEnvVar("registerId"))
```

My REQUEST 2 Pre Request has

```
console.log("registerId: "+bru.getEnvVar("registerId"))
console.log("registerId2: "+"{{registerId}}")
```

And I can see this working if I run the request from the GUI.  I can see my registerID env variable is accessible through {{registerId}} from REQUEST2 as long as I am using the GUI.

If I run it from CLI, the console log shows that aparently registerID gets updated, but then I cannot access the updated env variabee from different request. 

bru run 'request1.bru' --env 'MY ENV'  --> registerId set to : XXXX-XXXX-XXXX
bru run 'request2.bru' --env 'MY ENV'  -->  registerId: [VALUE TAKEN FROM GUI, not CLI ]   registerId2: {{registerId}}

Any help much appreciated

**Comment by nbari** [comment]
hi @emartinezagra did you manage to fix the problem? I want to call an existing request, and parse the JSON response to extract only some files I want to use in the body for a POST request, any idea how to achieve this?
